### PR TITLE
testing adding employees

### PR DESCRIPTION
**Pull request by Joaoviana**

### CODE REVIEW

File: server/queries/readSurvey.js [hunk 1]
```diff
@@ -9,25 +9,32 @@ const readSurvey = async (_id) => {
     const survey = await surveys.findOne({
       _id: ObjectID(_id),
     });
+    console.log('SURVEY', survey);
```
Comment (marthaklambert) [inline server/queries/readSurvey.js]: some console.log removal when you have time

File: client/src/pages/EditSurvey/index.jsx [hunk 2]
```diff
@@ -91,14 +100,19 @@ const EditSurvey = ({ match, history }) => {
       // remove _id key from data to edit on survey; This key is immutable.
       // eslint-disable-next-line dot-notation
       delete editedSurvey['_id'];
+      delete editedSurvey.modalStyle;
+      delete editedSurvey.openModal;
+      delete editedSurvey.isConfirming;
+      console.log(editedSurvey);
```
Comment (AntonioGargaro) [inline client/src/pages/EditSurvey/index.jsx]: CONSOLE LOG

**Comment by TKostrzewski** [comment]
We could add an exit button in the top right corner of the "Add Recipients" modal? To exit without saving changes, I have to click outside of the modal.

File: client/src/pages/CreateSurvey/index.jsx [hunk 3]
```diff
@@ -43,10 +43,13 @@ const CreateSurvey = ({ history }) => {
     modalStyle,
     isConfirming,
   } = useSelector((state) => state.createSurveyReducer);
+
+  const { recipientIds } = useSelector((state) => state.employeeTableReducer);
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch({ type: 'RESET_CS_MODAL_STATE' });
```
Comment (TKostrzewski) [inline client/src/pages/CreateSurvey/index.jsx]: CS (in the type) isn't clear to me what it means.

File: client/src/pages/EditSurvey/index.jsx [hunk 4]
```diff
@@ -78,7 +87,7 @@ const EditSurvey = ({ match, history }) => {
   }, [match.params, dispatch]);
 
   const toggleModal = () => {
-    dispatch({ type: 'TOGGLE_MODAL' });
+    dispatch({ type: 'TOGGLE_ES_MODAL' });
```
Comment (TKostrzewski) [inline client/src/pages/EditSurvey/index.jsx]: ES is not clear to me what it stands for.

File: client/src/pages/CreateSurvey/index.jsx (hunk 3 above)
Comment (arrested-developer) [reply client/src/pages/CreateSurvey/index.jsx]: This is a very useful review comment @TKostrzewski ! I don't know what this means either. Verbose variable / constant names are perfectly acceptable

**Pull request merged**